src/stories/MatrixVis.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Story } from '@storybook/react/types-6-0';
+import ndarray from 'ndarray';
+import type { H5WebComplex } from '../h5web/providers/models';
+import { formatNumber } from '../h5web/vis-packs/core/utils';
+import {
+  getMockDataArray,
+  MatrixVis,
+  MatrixVisProps,
+  mockValues,
+} from '../packages/lib';
+import FillHeight from './decorators/FillHeight';
+
+const Template: Story<MatrixVisProps> = (args) => <MatrixVis {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  dataArray: getMockDataArray('/nD_datasets/twoD'),
+  formatter: (val) => formatNumber(val as number),
+  cellWidth: 100,
+};
+
+export const Complex = Template.bind({});
+
+Complex.args = {
+  dataArray: ndarray(mockValues.twoD_cplx.flat(1) as H5WebComplex[], [2, 2]),
```

Comment:
`getMockDataArray('/nD_datasets/twoD_cplx')` does not work ?

Reply:
You convinced me to have another go. I made it work with a generic.